index.js

```diff
@@ -25,6 +25,11 @@ function getUUID() {
   return bot._client.session.selectedProfile.id;
 }
 
+function setlang() {
+  logger.info('Changing language.');
+  bot._client.write('chat', { message: '/lang ' + config.CHANGE_LANGUAGE });
```

Comment:
You can just use `bot.chat()` here

Reply:
I see, I was attempting to make it somewhat resemble the format of the code.